docs/running-on-kubernetes.md

```diff
@@ -990,6 +990,52 @@ See the [configuration page](configuration.html) for information on Spark config
   Specify whether executor pods should be deleted in case of failure or normal termination.
   </td>
 </tr>
+<tr>
+  <td><code>spark.kubernetes.kubernetesClient.submission.connectionTimeout</code></td>
+  <td>10000</td>
```

Comment:
BTW, is there any reason to choose `10000` here?

Reply:
yea that is the current default of the underlying okhttp client.

Reply:
Can we simplify names of those new options by removing `kubernetesClient`?